websites/Z/Zee5/presence.ts

```diff
@@ -0,0 +1,85 @@
+const presence = new Presence({
+		clientId: "1103408220921397349",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+enum Assets {
+	Logo = "https://i.imgur.com/rMcEfBA.jpg",
+	Play = "https://i.imgur.com/lytENvp.png",
+	Pause = "https://i.imgur.com/NT77akx.png",
```

Comment:
now just use the Play and Pause assets from the index.d.ts, you don't need to define them here.
You might have to change at line 6 to const enum